mcache/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,74 @@
+name: Memcached
+files:
+- name: mcache.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: url
```

Comment:
```suggestion
    - name: url
      required: true
```

Reply:
@ChristineTChen  Technically user can choose between `url` and `socket`, so `url` is not required

Reply:
ah makes sense, I saw the old `required` param and assumed it was a discrepancy.